### ISSUE TITLE
Cannot evaluate inp query

### ISSUE THREAD

**Issue by slnxyr** [LHRLAB/NQE#2]
Hi, I follow your guide to train and evaluate the model by 1p-train, but there is something wrong when evaluate inp query. Could you help me?  Thanks.
Beside, how long does it take to train all queries？
![image](https://github.com/LHRLAB/NQE/assets/53633782/7f7f4cf2-d3b2-4f27-b078-400b49809d87)

**Comment by LHRLAB** [comment]
It's strange that you may have a problem with the version of the package.

**Comment by slnxyr** [comment]
But the other queries can get the evaluation result in the same code.

**Comment by LHRLAB** [comment]
That may be the format problem of the inp data set. We will check it, and you can try to download the data set again.

**Comment by YiZuMonash** [comment]
Same question as @slnxyr , I ran the code based on the full queries, but it seems to take a long time. May I know how long will it takes because it still in epoch 1 for nearly 1 day which confused me a lot. my settings as follows. Thanks.

parser.add_argument("--train_tasks", default="1p,2p,3p,2i,3i,pi,ip,2u,up,2cp,3cp,2in,3in,inp,pin,pni", type=str)

**Comment by LHRLAB** [comment]
Yes, full-query training does take a long time, but because NQE is based on Transformer, you can achieve a similar effect of full-query training with only 1p training.
